pipelines.adoc

```diff
@@ -73,13 +67,14 @@ Alternatively, if you are using an unmanaged cluster, you can set up a persisten
 . Clone the link:https://github.com/kabanero-io/kabanero-pipelines[kabanero-pipeplines repo]
 
   git clone https://github.com/kabanero-io/kabanero-pipelines
+  
+===== Static Persistent Volumes
 
-. Clone the `pv.yaml` file in this repo and apply it.
-
+If you are not running your openshift cluster on public cloud, you can setup the persistent volume statically. For example on the usage of static persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#static-persistent-volumes[here]
 
- cd ./pipelines/common
- oc apply -f pv.yaml -n kabanero
+===== Dynamic Volume Provisioning
 
+If you are running your openshift cluster on a public cloud, you can take advantage of dynamically provisioning the persistent volume.You can use your cloud provider’s default storage class for it. For example on the usage of dynamic persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#dynamic-volume-provisioning[here]
```

Comment:
**change**
If you are running your openshift cluster on a public cloud, you can take advantage of dynamically provisioning the persistent volume.You can use your cloud provider’s default storage class for it. For example on the usage of dynamic persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#dynamic-volume-provisioning[here].

**to**
If you run your OpenShift cluster on a public cloud, you can set up a dynamic persistent volume by using  your cloud provider’s default storage class. For an example of how to use dynamic persistent volume provisioning, see https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#dynamic-volume-provisioning[Dynamic Volume Provisioning].

Reply:
looks good